secant/Features/WalletEventsFlow/Views/TransactionDetailView.swift

```diff
@@ -96,7 +96,7 @@ extension TransactionDetailView {
 
     func address(mark: RowMark = .neutral, viewStore: WalletEventsFlowViewStore) -> some View {
         Button {
-            viewStore.send(.copyToPastboard(transaction.address))
+            viewStore.send(.copyToPastboard(transaction.address.redacted))
```

Comment:
Wouldn't this make the clipboard to have a redacted version of the address?

Reply:
Short answer: no

Long one: it only creates a redacted struct of the string so any attempt to dump it will be redacted. The value inside is still just a String, all unit tests passing. We don't use description of customDump to pass values around, it's only a protection from the logs.